src/codegen/mod.rs

```diff
@@ -2578,9 +2578,9 @@ impl CodeGenerator for Enum {
 
         // TODO(emilio): Delegate this to the builders?
         match variation {
-            EnumVariation::Rust(non_exhaustive) => {
+            EnumVariation::Rust { non_exhaustive: nh } => {
```

Comment:
`EnumVariation::Rust { non_exhaustive } => {`

Then just use `non_exhaustive` instead of nh?